--- provision_cf.ori	2015-03-31 15:38:15.975328901 +0000
+++ provision_cf	2015-03-31 15:38:33.163921037 +0000
@@ -43,8 +43,9 @@
 }
 
 get_ip_from_vagrant_ssh_config() {
-  config=$(vagrant ssh-config)
-  echo $(echo "$config" | grep HostName | awk '{print $2}')
+#  config=$(vagrant ssh-config)
+#  echo $(echo "$config" | grep HostName | awk '{print $2}')
+   echo 192.168.50.4
 }
 
 get_ip_from_vm_ifconfig() {
